Containerfile

```
@@ -20,9 +22,14 @@ COPY pyproject.toml uv.lock ${PROJ_DIR}
 RUN --mount=type=cache,target=/root/.cache/uv \
     uv sync
 
-# Install Node.js dependencies
-COPY package*.json ${PROJ_DIR}
-RUN npm install --no-color
+# Install pnpm globally
+RUN npm install -g pnpm@latest-10
+
+# Install Node.js dependencies with pnpm
+# Uses a persistent cache mount for pnpm (see https://pnpm.io/docker#minimizing-docker-image-size-and-build-time)
+COPY package.json pnpm-lock.yaml ${PROJ_DIR}
+RUN --mount=type=cache,id=pnpm,target=${PNPM_HOME}/store \
+    pnpm install --frozen-lockfile
```

Can you make this:

```suggestion
    pnpm install --frozen-lockfile --prod
```

This will require moving `webpack` out of the dev dependencies and into the main dependencies which is a change I've been wanting to make for a little while.

I don't *think* we need ESLint in the container. I only use ESLint locally, but if you think there's a use case for linting in the container, maybe we can install all the dev dependencies, too.

But still, I think Webpack needs to be a prod dependency.